charts/console/templates/deployment.yaml

```diff
@@ -271,6 +264,14 @@ spec:
             - name: REDPANDA_ADMINAPI_TLS_CERTFILEPATH
               value: "/etc/console/secrets/redpanda-admin-api-tls-cert"
             {{- end }}
+            {{- else }}
```

Comment:
could you fix the indentation to make it clear where the logic block start/end points are?

Reply:
I will only fix the relevant pieces for now.